packages/create-cloudflare/templates-experimental/hello-world-durable-object-with-assets/js/__dot__gitignore

```diff
@@ -0,0 +1,172 @@
+# Logs
+
+logs
+_.log
+npm-debug.log_
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
+
```

Comment:
I just notice that these `__dot_gitignore` file should be ignored. There is already an entry to ignore all of these on `templates` so you probably need to add one more for the `templates-experimental`:

```sh
# Already in .gitignore
templates/**/__dot__gitignore
# I think we should add this
templates-experimental/**/__dot__gitignore
```

We might have it added with other experimental templates too. So it would be great if you can remove them as well. 👍🏼